code/game/gamemodes/endgame/xmas/victorianstation.dm

```diff
@@ -0,0 +1,106 @@
+/**
+	Victorianize your station
+		At one press of a button, turn every wall on the station to wood, all lighting to lanterns, all lamps to candles,
+		And the main hallways to snow.
+
+		Additionally, the bar becomes much more festive, with the addition of a christmas tree!
+
+		Also piss off your server host and set the entire server on fire.
+*/
+
+/datum/universal_state/auldlangsyne
+	name = "Older times"
+	desc = "The clock's ticking backwards!"
+
+
+/datum/universal_state/auldlangsyne/OnEnter()
+	var/target_zlevel = map.zMainStation
+
+	to_chat(target_zlevel, "<span class='sinister'>There is a certain chill to the air, as bells ring faintly in the distance...</span>")
+	//Snow up the halls
+	for(var/A in typesof(/area/hallway))
+		var/area/to_snow = locate(A)
+		if(to_snow)
+			for(var/turf/simulated/floor/F in to_snow)
+				new /obj/structure/snow(F)
+				for(var/cdir in cardinal)
+					var/turf/TT = get_step(F,cdir)
+					if(istype(TT,/turf/simulated/wall))
+						new/obj/machinery/xmas_light(TT,cdir)
+			for(var/obj/machinery/light/L in to_snow)
+				qdel(L)
+
+	for(var/area/A in areas)
+		if(A.z == target_zlevel)
+			for(var/turf/T in A)
+				if(istype(T, /turf/simulated/wall) && !istype(T, /turf/simulated/wall/r_wall))
+					T.ChangeTurf(/turf/simulated/wall/mineral/wood, tell_universe = 0)
+			for(var/obj/item/device/flashlight/F in A)
+				var/obj/item/candle/C = new /obj/item/candle(F.loc)
+				C.light(quiet = 1)
+				qdel(F)
+			for(var/obj/machinery/light/L in A)
+				var/turf/T = get_turf(L)
+				var/obj/structure/hanging_lantern/HL = new /obj/structure/hanging_lantern/dim(T)
+				HL.dir = L.dir
+				HL.update()
+				qdel(L)
+			for(var/obj/structure/closet/secure_closet/S in A)
+				switch(S.type)
+					if(/obj/structure/closet/secure_closet/captains)
+						new /obj/item/clothing/suit/wintercoat/captain(S)
```

Comment:
These spawns like this won't work properly if the locker's open.

Reply:
you then close it, and open it again.

Magic of christmas!

(Can't find anything that can help 'place' something in a locker)